frontend/test/metabase/scenarios/question/caching.cy.spec.js

```diff
@@ -2,9 +2,11 @@ import {
   restore,
   describeEE,
   mockSessionProperty,
-  modal,
   visitQuestion,
+  questionInfoButton,
+  rightSidebar,
 } from "__support__/e2e/cypress";
+import { popover } from "../../../__support__/e2e/helpers/e2e-ui-elements-helpers";
```

Comment:
It should probably be an absolute import: `import { popover } from "__support__/e2e/helpers/e2e-ui-elements-helpers`